src/requests.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+export default function submitRegistrationData(data) {
+  return axios.post('localhost:3210/users/register', {
+    firstName: data.firstName,
+    lastName: data.lastName,
+    password: data.password,
+    email: data.email,
+    birthDate: data.birthDate,
+    job: data.job,
+    gender: data.gender,
+    street: data.street,
+    zipCode: data.zipCode,
+    country: data.country
+  });
```

Comment:
Looks fine but please use object destructuring

Reply:
I had to change eslint rule for object-curly-newline ("multiline": true instead of never) to complete it